source/mainnet/net/mobile-wallet/update-baker-mw.rst

```diff
@@ -273,50 +273,54 @@ Update pool settings
 
     #. Click |earn| on the navigation bar. Use the left and right arrow to locate the button if necessary. Use the Accounts drop-down at the top of the window to select the account.
 
-        .. image:: ../images/browser-wallet/setup-baking-delegation.png
-            :width: 40%
+       .. image:: ../images/browser-wallet/setup-baking-delegation.png
+           :width: 40%
 
     #. You see the current baker setup. Click **Update**.
 
-        .. image:: ../images/browser-wallet/update-baker-options.png
-            :width: 40%
-            :alt: screen showing current baker setup and buttons to stop or update
+       .. image:: ../images/browser-wallet/update-baker-options.png
+           :width: 40%
+           :alt: screen showing current baker setup and buttons to stop or update
 
     #. Click **Update pool settings**. Review the information about updating baker pool settings and click **Continue**.
 
-        .. image:: ../images/browser-wallet/update-baker-actions.png
-            :width: 40%
-            :alt: screen showing actions to update baker
+       .. image:: ../images/browser-wallet/update-baker-actions.png
+           :width: 40%
+           :alt: screen showing actions to update baker
 
     #. You can change the baker pool status, if desired. Click **Continue**. The options are:
 
-        - Open pool: open a pool for a previously closed baker
-        - Closed for new: close the pool to new delegators. Existing delegators are not affected. You might do this when, for example, the pool is close to meeting one or both of the :ref:`bounding caps<delegation-concept>`. **It is the baker’s responsibility to monitor the stake to make pool management decisions.**
-        - Close pool: close a pool permanently.
+       - Open pool: open a pool for a previously closed baker
+       - Closed for new: close the pool to new delegators. Existing delegators are not affected. You might do this when, for example, the pool is close to meeting one or both of the :ref:`bounding caps<delegation-concept>`. **It is the baker’s responsibility to monitor the stake to make pool management decisions.**
+       - Close pool: close a pool permanently.
```

Comment:
I guess this will be translated in the main tokenomics update. :thinking:

Reply:
Yep